.editorconfig

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
next time, please focus on feature branches. `.editorconfig` is totally fine, just dont add it as part of something else